### ISSUE TITLE
Resolve pointcloud_octomap_updater TF errors during pick and place actions

### ISSUE THREAD

**Issue by alexander-sung** [DFKI-NI/mobipick_labs#5]
When the robot is performing pick or place actions, the following errors appear:

```
Transform error: Lookup would require extrapolation 1.600000000s into the future.  Requested time 966.000000000 but the latest data is at time 964.400000000, when looking up transform from frame [mobipick/structure_cover_back_link] to frame [mobipick/gripper_astra_depth_optical_frame] [/mobipick/move_group]
Transform cache was not updated. Self-filtering may fail. If transforms were not available yet, consider setting robot_description_planning/shape_transform_cache_lookup_wait_time to wait longer for transforms [/mobipick/move_group]
```

Interestingly, as long as these errors only appear once per action, they don't seem to report an actual error. The action is performed nevertheless correctly.

**Comment by mintar** [comment]
This seems to be a bug in the pointcloud_octomap_updater: https://github.com/ros-planning/moveit/issues/3242

We already increased `shape_transform_cache_lookup_wait_time` to 0.1 s, but that doesn't always help. I suspect that the TF cache inside the pointcloud_octomap_updater doesn't get updated regularly or something similar. Doesn't happen when using the depthimage_octomap_updater (on the real robot).

**Comment by mintar** [comment]
To be a bit more specific: I suspect that outside of pick and place actions, the `pointcloud_octomap_updater` TF cache gets updated from TF callbacks from the MoveIt main thread. When we run a pick and place action, the MoveIt main thread blocks and the `pointcloud_octomap_updater` TF cache doesn't get updated. This is just a hunch though.

Interestingly, this only happens during pick and place, and never during regular arm movements. And always exactly once.

**Comment by mintar** [comment]
Ok, I found a better way to fix this (see detailed commit message in 144194d).
